oxd-server/pom.xml

```diff
@@ -189,6 +190,32 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
```

Comment:
we don't want any installer generation related things here. If you really want to keep it mavenized, probably good idea to keep it in maven profile. So we can activate it when needed.

Reply:
Created "installer" profile for generating installer directory.